frame/example/src/benchmarking.rs

```diff
@@ -0,0 +1,40 @@
+#![cfg(feature = "runtime-benchmarks")]
+
+use crate::*;
+
+mod benchmarking {
```

Comment:
you shouldnt need another `mod` here. By making this file, you have already wrapped this logic into a module.

```suggestion
```